tools/cli/tests/end-to-end.py

```diff
@@ -58,6 +59,15 @@ def free_port():
     return port_number
 
 
+def random_zone():
+    zones_list = subprocess.check_output([
+        'gcloud', 'compute', 'zones', 'list',
+        '--filter=region~us-west', '--format=value(name)']).decode(
```

Comment:
Should this be `--format=value("name")`?

Reply:
No; the string is already in quotes, so nested quotes would be passed directly to the format handler, which would not know what to do with them.

Reply:
Ah, what threw me off is I didn't know this actually executed as 'gcloud' 'compute'... etc (with the quotes). It fails for me without the quotes.